client/blocks/signup-form/index.jsx

```diff
@@ -185,6 +188,16 @@ class SignupForm extends Component {
 		}
 	}
 
+	sanitizeName( name ) {
```

Comment:
This whole function is all sorts of wrong for internationalization reasons ...

Is there a reason we need to do this at all? If someone want's to say their name is: `J4min B*tz`, who's to say that it isn't? :)

I'd use a more Unicode example with Ás and such, but hopefully, you get the point. We can't predict what characters are in people's names. :)